homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -69,25 +67,25 @@
 
     hass.data.setdefault(DOMAIN, {})
     accesspoints = config.get(DOMAIN, [])
+
     for conf in accesspoints:
+        apid = conf[CONF_ACCESSPOINT].replace('-', '').upper()
+        if apid in hass.data[DOMAIN]:
```

Comment:
This can't happen.

Reply:
I you suggest to remove the code inside the loop and fire  'hass.async_add_job(hass.config_entries.flow.async_init(.....' and implement the 'async def async_step_import(self, import_info)' in config_flow.py?